MANIFEST.in

```diff
@@ -1,4 +1,3 @@
-include README.md
```

Comment:
I think being explicit is good in this case.

Reply:
Added README.md back.